examples/bri-3/src/bri/identity/bpiSubjects/agents/bpiSubjectsStorage.agent.ts

```diff
@@ -70,8 +71,37 @@ export class BpiSubjectStorageAgent extends PrismaService {
     );
   }
 
+  async storePublicKey(
+    id: string,
+    type: string,
+    value: string,
+    bpiSubjectId: string,
+  ): Promise<void> {
+    await this.prisma.publicKey.create({
+      data: {
+        id: id,
+        type: type,
+        value: value,
+        bpiSubjectId: bpiSubjectId,
+      },
+    });
+  }
+
+  async updatePublicKey(
+    type: string,
+    value: string,
+    bpiSubjectId: string,
+  ): Promise<PublicKey> {
+    const updatedPublicKey = await this.prisma.publicKey.update({
+      where: { type_bpiSubjectId: { type: type, bpiSubjectId: bpiSubjectId } },
+      data: {
+        value: value,
+      },
+    });
+
+    return this.mapper.mapPublicKeyPrismaModelToDomainObject(updatedPublicKey);
```

Comment:
Make sure to use the new mapping approach from main

Reply:
Done.